doc/guides/onboarding-extras.md

```diff
@@ -24,7 +24,17 @@ request.
 * `tsc-agenda` - Open issues and pull requests with this label will be added to
   the Technical Steering Committee meeting agenda
 
---
+---
+
+* `author-ready` - A pull request is _author ready_ when:
+  * There is a CI run in progress or completed.
+  * There is at least one Collaborator approval.
```

Comment:
```suggestion
  * There is at least one Collaborator approval (or two TSC approvals for semver-major PRs)
```

Reply:
As I understand it the `author ready` label does not mean the PR is ready to land. It means that there's nothing more for the author of the PR to do (i.e. no further changes required).

Reply:
The `author ready` label is supposed to indicate that it is ready to land, possibly pending some waiting time and re-running CI (this is why I originally named this `ready` instead of `author ready`). I would apply @jasnell’s suggestion.